docs/REST API/rest-api.md

```diff
@@ -11125,8 +10841,13 @@ Transforms a host name into a host id. For example, the Fleet UI use this endpoi
 - [Require password reset](#require-password-reset)
 - [List a user's sessions](#list-a-users-sessions)
 - [Delete a user's sessions](#delete-a-users-sessions)
+- [Create invite](#create-invite)
+- [List invites](#list-invites)
+- [Delete invite](#delete-invite)
+- [Verify invite](#verify-invite)
+- [Update invite](#update-invite)
```

Comment:
```suggestion
- [Modify invite](#modify-invite)
```

I think we want to use "Modify" instead of "Update" for all endpoints that modify/edit something.